File: code_sandbox/app_test/assets/mock_entry.js
function generate() {
  const a = 2;
  const b = 3;
  const total = a + b;

  console.log("mock js no import");

  return {
    source: "mock_entry.js",
    values: { a: a, b: b },
    total: total,
  };
}

module.exports = { generate };

File: code_sandbox/app_test/assets/mock_entry.py
from typing import Dict, Any


def generate() -> Dict[str, Any]:
    a = 2
    b = 3
    total = a + b

    print("mock py no import")

    return {
        "source": "mock_entry.py",
        "values": {"a": a, "b": b},
        "total": total,
    }
